.github/workflows/verify_consumer_pacts.yaml

```diff
@@ -90,20 +115,23 @@ jobs:
       - name: Extract branch
         id: extract-branch
         run: |
-          GITHUB_EVENT_NAME=${{ github.event_name }}
-          if [[ "$GITHUB_EVENT_NAME" == "push" || "$GITHUB_EVENT_NAME" == "workflow_dispatch"]]; then
-            GITHUB_REF=${{ github.ref }} # The Git Ref that this workflow runs on
-            GITHUB_SHA=${{ github.sha }} # The Git Sha that this workflow runs on
+          if [[ "$GITHUB_EVENT_NAME" == "push" ]]; then
+            GITHUB_REF=${{ github.ref }}
+            GITHUB_SHA=${{ github.sha }}
           elif [[ "$GITHUB_EVENT_NAME" == "pull_request" ]]; then
             GITHUB_REF=refs/heads/${{ github.head_ref }}
             GITHUB_SHA=${{ github.event.pull_request.head.sha }}
+          elif [[ "$GITHUB_EVENT_NAME" == "workflow_dispatch" ]]; then
```

Comment:
I got previous feedback to collapse this conditional block with the one for push since they are identical, however I introduced a syntax error when doing that, and put it back to the way it was before to also make it more identical to the upstream PR from which I copied this stuff.  I'm hoping that will make convergence easier, at the expense of a little more duplication for the time being.

Reply:
Good to go on this one. The `push` and `workflow_dispatch` conditions have been collapsed into one condition.